packages/camera/camera_web/example/integration_test/camera_service_test.dart

```diff
@@ -102,7 +102,6 @@ void main() {
           expect(
             () => cameraService.getMediaStreamForOptions(
               const CameraOptions(),
-              cameraId: cameraId,
```

Comment:
cc @stuartmorgan 

So this lint warning is a little weird. The default value for the parameter is `0` and the `cameraId` variable is set to `0`. However, there are other locations in the test that use the `cameraId` variable. So, if the variable was changed, many of the tests will fail. I think we should actually just change the variable to `1` instead to avoid this problem.

Reply:
fixed in [38eaffa](https://github.com/flutter/plugins/pull/6163/commits/38eaffa10e34d442ad98669b697221ea513615a4)